server/sonar-web/src/main/js/apps/organizations/components/MembersListItem.js

```diff
@@ -40,14 +45,41 @@ export default class MembersListItem extends React.PureComponent {
     return (
       <tr>
         <td className="thin nowrap">
-          <Avatar hash={member.avatar} size={AVATAR_SIZE} />
+          <Avatar hash={member.avatar} email={member.email} size={AVATAR_SIZE} />
         </td>
         <td className="nowrap text-middle"><strong>{member.name}</strong></td>
```

Comment:
user login would be useful here for the same reason why we display it when adding a new member